bitcoin-design/learning-bitcoin-and-design/fedimint.md

```diff
@@ -452,152 +445,155 @@ But basically you convince them, hey, I'm me and you have my shard of my seed an
+And then from an ecosystem point of view, I think it would be very interesting to see a diversity of deployments and then in the wild, discover what are the best ways of using it,, through testing, with small amounts of money and in different places, like, you can actually figure out through testing what people like rather than, having to just guess and build your one system and hope it's right.
 
 ## Backing up other user wallet metadata
 
-Speaker 0: 00:38:44
+Stephen DeLorme: 00:38:44
 
-Tristan, you had a comment.
+Triston, you had a comment.
 
-Speaker 5: 00:38:47
+Triston: 00:38:47
 
 I just didn't have a comment.
 Yes, so I was wondering if with this architecture there would be any kind of like data store.
-And I guess like as I say that I realized that there might be some privacy drawbacks of doing so and the reason that I'm asking about data storage because there's some additional kind of information that a wallet may need to store, for example, labels and contact data and whatever within the Mint.
-And that could even be stored on, as you mentioned, something like Google Cloud or iCloud if it's using something like Photon, but yeah, there's just like a mass amount of like arbitrary data that someone's wallet within the Mint might want to store.
+And I guess like as I say that I realized that there might be some privacy drawbacks of doing so and the reason that I'm asking about data storage because there's some additional kind of information that a wallet may need to store, for example, labels and contact data and whatever within the mint.
+And that could even be stored on, as you mentioned, something like Google Cloud or iCloud if it's using something like Photon, but yeah, there's just like a mass amount of like arbitrary data that someone's wallet within the mint might want to store.
 Settings, for example, different configurations.
 Is something like that being considered or is it just going to be like left for like a plugin based system?
 
-Speaker 2: 00:40:01
+Justin Moon: 00:40:01
 
-Yeah so I mean on the server side like each server has a database And that's what basically the what the federation member, the servers send messages back and forth is basically to just ensure they have exactly synchronized database.
+Yeah so I mean on the server side like each server has a database.
+And that's what basically the what the federation member, the servers send messages back and forth is basically to just ensure they have exactly synchronized database.
 And so with the, these recovery ideas, like sending a shard of your seed to one of these servers, it would just go on their database.
 Right.
```

Comment:
This kind of "Right" in the middle of a speaker's segment is most times something that another speaker will say and it will be attributed to the current speaker. Feel free to remove those as their are non-essential lines.